src/sessions.ts

```diff
@@ -72,6 +75,15 @@ export type ClientSessionEvents = {
 
 /** @internal */
 const kServerSession = Symbol('serverSession');
+/** @internal */
+const kPinnedConnection = Symbol('pinnedConnection');
+
+/** @public */
+export interface EndSessionOptions {
+  /** An optional error which caused the call to end this session */
```

Comment:
Is this only for our use? Can mark as `@internal` if so.

Reply:
The problem is that it's part of the public API for `endSession`, and api-extractor would not let me mark individual signatures of that method internal and some public. Or maybe I'm missing how to do that?

Reply:
```ts
/** @public */
export interface EndSessionOptions {
  /**
   * An optional error which caused the call to end this session
   * @internal
   */
  error?: AnyError;
  force?: boolean;
}
```
produces:
```ts
/** @public */
export declare interface EndSessionOptions {
    /* Excluded from this release type: error */
    force?: boolean;
}

```

So just on the error property will do

Reply:
Thanks - updated.